site/src/atomics/flex.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Flex Atomics
+description: Flex related atomic css classes for the Atlas Design System
+template: standard
+---
+
+# Flex Atomics
+
+Flex atomics are useful when composing layouts and patterns containing other components.
+
+| cssproperty       | value                                                                               | screensize |
+| ----------------- | ----------------------------------------------------------------------------------- | ---------- |
+| `flex-direction`  | `column`, `row-reverse`, `column-reverse`                                           | `tablet`   |
+| `justify-content` | `flex-start`, `flex-end`, `center`, `space-around`, `space-between`, `space-evenly` | `tablet`   |
+| `align-items`     | `flex-start`, `flex-end`, `center`, `baseline`, `stretch`                           | `tablet`   |
+| `align-self`      | `flex-start`, `flex-end`, `center`, `baseline`, `stretch`                           | `tablet`   |
+| `flex-wrap`       | `flex-wrap-wrap`, `flex-wrap-wrap-reverse`                                          | `tablet`   |
```

Comment:
This line should include nowrap (just went in to main branch) wrap, and wrap-reverse.

Reply:
added